ssclj/jar/src/com/sixsq/slipstream/ssclj/resources/job/utils.clj

```diff
@@ -1,11 +1,9 @@
 (ns com.sixsq.slipstream.ssclj.resources.job.utils
   (:require
-    [com.sixsq.slipstream.db.impl :as db]
-    [com.sixsq.slipstream.ssclj.middleware.cimi-params.utils :as cimi-params-utils]
-    [com.sixsq.slipstream.ssclj.filter.parser :as parser]
+    [clj-time.core :as time]
     [clojure.tools.logging :as log]
     [com.sixsq.slipstream.ssclj.resources.common.utils :as u]
-    [clj-time.core :as time]))
+    [com.sixsq.slipstream.ssclj.util.zookeeper :as uzk]))
 
 (defn stop [{state :state id :id :as job}]
   (if (= state "RUNNING")
```

Comment:
- Is the logging useful here?  The state changes for the other states are not logged.